content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,237 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
+homepage by entering the :abbr:`IP (Internet Protocol)` address of the :abbr:`IoT (Internet of
+Things)` box into a browser window from a computer on the same network as the :abbr:`IoT (Internet
+of Things)` box (preferably by ethernet connection).
+
+.. note::
+   The :abbr:`IP (Internet Protocol)` address can be accessed by the router admin console that the
+   :abbr:`IoT (Internet of Things)` box is connected to, or by connecting a working receipt printer
+   to the :abbr:`IoT (Internet of Things)` box. A receipt will print out with the :abbr:`IoT
+   (Internet of Things)` box's :abbr:`IP (Internet Protocol)` address on it.
+
+Then click on the :guilabel:`Printers server` button. This will redirect the browser to the *CUPS
+page*. Next, go to :menuselection:`Administration --> Printers --> Add Printer`. Now choose the
+printer that should be modified. It could be listed as :guilabel:`Unknown`. Click
+:guilabel:`Continue`.
+
+.. tip::
+   If the name of the printer is still uncertain, take the following steps:
+
+   #. Take note of the listed printers on the *CUPS* page.
+   #. Turn the printer off and refresh the page.
+   #. Now compare the difference with the first list to see which printer disappeared.
+   #. Turn the printer back on and refresh the page again.
+   #. Double check the list again to see if the printer re-appears.
+   #. The printer that disappeared and reappears again on the listed printers in the name of the
+      printer in question.
+
+   .. note::
+      This can be :guilabel:`Unknown` under local printers.
+
+`CUPS` will prompt the admin for three pieces of information: the :guilabel:`Name`,
+:guilabel:`Description` and the :guilabel:`Location`. The last to pieces of information do not need
+to be specific, however, the :guilabel:`Name` should follow a particular convention to work with the
+`ESC *` command.
+
+The :guilabel:`Name` should match this convention:
+`<printer_name>__IMC_<param_1>_<param_2>_..._<param_n>__`
+
+A breakdown of the naming convention:
+
+- `printer_name`: Is the printer name. It can be any character as long as it does not contain `_`,
+  `/`, `#`, or ` ` (space character).
+- `IMC`: Stands for Image Mode Column (the simplified name for `ESC *`).
+- `param_1`: specific parameter:
+
+  - `SCALE<X>`: Scale of the picture (with the same aspect ratio). `X` should be an integer
+    describing the scale percentage that should be used.
+
+    .. example::
+       `100` is the original size, `50` is half the size, `200` is twice the size.
+
+  - `LDV`: Low Density Vertical (will be set to High Density Vertical if not specified).
+  - `LDH`: Low Density Horizontal (will be set to High Density Horizontal if not specified).
+
+    .. note::
+       "Density" parameters might need to be configured in a particular way depending on the printer
+       model.
+
+.. seealso::
+   Visit `Epson's documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ and click on the
```

Comment:
same as above, consider changing it to be more explicit: "Epson's ESC * documentation"

Reply:
Noted, I've made a change.